README.md

```diff
@@ -1,43 +1,69 @@
-COSI repos and images:
+![version](https://img.shields.io/badge/status-pre--alpha-lightgrey)
 
-[Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec)  \
-[API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) \
-[Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) <br/>
-&emsp; - [images: cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) \
- [Provisioner Sidecar](https://github.com/kubernetes-sigs/container-object-storage-interface-provisioner-sidecar) <br /> 
- &emsp; -  images: cosi-provisioner \
- [CSI Adapter](https://github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter) <br /> 
- &emsp; - images: cosi-node-adapter 
+# Container Object Storage Interface Spec
 
-COSI deprecated repos:   
+This repository hosts the gRPC API for the Container Object Storage Interface (COSI) standard. The interfaces defined in the [gRPC specification](cosi.proto) are meant to be the common interface for object storage provisioning and management across various object storage vendors.
 
+For more information about the COSI effort, visit our [documentation](https://github.com/kubernetes-sigs/container-object-storage-interface-api/tree/master/docs/index.md).
 
+## Why another standard?
 
-[Spec](https://github.com/container-object-storage-interface/spec) \
-[API](https://github.com/container-object-storage-interface/api) \
-[Manager (i.e. Controller)](https://github.com/container-object-storage-interface/cosi-controller-manager) \
-[Provisioner Sidecar](https://github.com/container-object-storage-interface/cosi-provisioner-sidecar) \
-[Ephemeral CSI Driver](https://github.com/container-object-storage-interface/ephemeral-csi-driver) 
+Kubernetes abstracts file/block storage via the CSI standard. The primitives for file/block storage do not extend well to object storage. Here is the **_extremely_** concise and incomplete list of reasons why:
 
+ - Unit of provisioned storage - Bucket instead of filesystem mount or blockdevice.
+ - Access is over the network instead of local POSIX calls.
+ - No common protocol for consumption across various implementations of object storage. 
+ - Management policies and primitives - for instance, mounting and unmounting do not apply to object storage.
 
+The existing primitives in CSI do not apply to objectstorage. Thus the need for a new standard to automate the management of objectstorage.
 
-# Container Object Storage Specification
+## Developer Guide
 
-Kubernetes specific Container Object Storage Interface (COSI) components.
+All API definitions **_MUST_** satisfy the following requirements:
 
-## Community, discussion, contribution, and support
+ - Must be backwards compatible
```

Comment:
I'm not sure if this is practical for alpha APIs?  In Kubernetes, if API stays in alpha, it does not need to support backward compatibility; but once it reaches beta, it has to support backward compatibility moving forward.

Reply:
This is a practical guideline for the community contributors who want to start contributing. I think between us in the team, we can have an understanding about it until we reach v1 status

Reply:
I think the language in the spec repo is supposed to be precise.  We can't require alpha APIs to be backward compatible.  Alpha means it is experimental.  If this requirement is added here but we are not really going to enforce it, then we should either loosen the requirement or remove it.

Reply:
makes sense. Ive updated the docs